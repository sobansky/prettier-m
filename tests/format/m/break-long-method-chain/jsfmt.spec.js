run_spec(__dirname, ["babel", "babel-flow", "flow", "typescript"], {
    breakLongMethodChains: true,
});