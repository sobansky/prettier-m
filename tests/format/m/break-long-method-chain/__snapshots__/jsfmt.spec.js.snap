// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`long-method-chain.js - {"breakLongMethodChains":true} format 1`] = `
====================================options=====================================
breakLongMethodChains: true
parsers: ["babel", "babel-flow", "flow", "typescript"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
obj.first(a).second(b).third(c);
=====================================output=====================================
obj
.first(a)
.second(b)
.third(c);

================================================================================
`;
